testhelpersterminal_reader.go

```diff
@@ -0,0 +1,22 @@
+//
```

Comment:
nit: could we add a "_" in the file name between "testhelpers" and "terminal_reader"? or is this the convention?

Reply:
From looking at the code this pattern seems to be very intentional.